ghec-audit-log-utils.js

```diff
@@ -15,7 +15,7 @@ function validateInput (program, config) {
   }
 
   // Validate correctness
-  const alphanumericRegex = /^[a-z0-9]+$/i
+  const tokenRegex = /^[A-Za-z0-9_]+$/i
```

Comment:
```suggestion
  const tokenRegex = /^[A-Za-z0-9_]+$/
```

As we're including capital- and lowercase in the regex, we don't need the `i` any longer.

Reply:
```suggestion
  const tokenRegex = /^[g(p|o|u|s|r)1_]{0,1}[A-Za-z0-9_]+$/
```

Alternatively to incorporate

- The character set is changing from `[a-f0-9]` to `[A-Za-z0-9_]`
- The format is changing to include a prefix:
  - `gp1_` for Personal Access Tokens
  - `go1_` for OAuth Access tokens
  - `gu1_` for GitHub App user-to-server tokens
  - `gs1_` for GitHub App server-to-server tokens
  - `gr1_` for GitHub App refresh tokens

Reply:
I like that suggestion, this way we ensure the format 👍

Reply:
Note: `{0,1}` is only there for backwards compatibility.